### PR TITLE
Update page in Connect API Manager “gateway-administation”

### PR DESCRIPTION
**Pull request by jcollins-axway**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by jcollins-axway** [comment]
JIRA APIGOV-15742 and APIGOV-15735

**Comment by jcollins-axway** [comment]
@kflan-io @dfeldick @sbolosan - please review for content

File: content/en/docs/central/connect-api-manager/gateway-administation.md [hunk 1]
```diff
@@ -150,9 +150,9 @@ This section connects the agent to AMPLIFY Central and determines how to publish
 
 `platformURL:` The AMPLIFY platform url. Needed for finding the user email during the subscription process with email notification. Default value is **<https://platform.axway.com>**.
 
-`teamID`: The Team identifier in AMPLIFY Central that all APIs will be linked. Locate this at AMPLIFY Central > Access > Team Assets. Open the teams details. The team identifier is the last part of the url (AMPLIFY URL/access/teams/detail/**e4ec6c1a69fd0b8e016a0bb0681e0e8f**).
+`team`: The Team name in AMPLIFY Central that all APIs will be linked. Locate this at AMPLIFY Central > Access > Team Assets.).
```
Comment (sbolosan) [inline content/en/docs/central/connect-api-manager/gateway-administation.md:153]: ... that all API's will be linked to.

Comment (jcollins-axway) [reply content/en/docs/central/connect-api-manager/gateway-administation.md]: fixed

**Pull request merged**